payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressSpec.kt

```diff
@@ -1,26 +1,47 @@
 package com.stripe.android.ui.core.elements
 
+import android.os.Parcelable
 import androidx.annotation.RestrictTo
 import com.stripe.android.ui.core.R
 import com.stripe.android.ui.core.address.AddressFieldElementRepository
 import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.Serializable
 
+@Serializable
 @Parcelize
+@Suppress("EnumEntryName")
+enum class DisplayField : Parcelable {
+    country
```

Comment:
nit: capitalized?

Reply:
The capitalization and underscores in variable names is all to match the JSon serialization.   I could go back and change them all to use the SerialName annotation though.

Reply:
I think we should do that, it makes it clearer and consistent